lib/libconfig.js

```diff
@@ -31,7 +31,14 @@ var envDefaults = {
 
   cors: false,
   body_parser_limit: false,
-  ldap: false
+  ldap: false,
+
+  jobsQuantityOnPage: {
```

Comment:
we need some documentation on this feature, in readme or somewhere

Reply:
5bd157e764b3dcfd0f8f2f71a04fba019d592e19